src/pages/HomePage/Componets/index.jsx

```diff
@@ -0,0 +1,11 @@
+export * from "./Hero/Hero";
+export * from "./AboutUs/AboutUs";
+export * from "./Dashboard/Dashboard";
+export * from "./OurServices/OurServices";
+export * from "./OurCustomers/index";
+
+export * from "./Hero/Hero.css";
+export * from "./AboutUs/AboutUs.css";
+export * from "./Dashboard/Dashboard.css";
+export * from "./OurServices/OurServices.css";
+export * from "./OurCustomers/OurCustomers.css";
```

Comment:
css should be used inside of the components. No need to export them. They should imported **only** in corresponding files. E.g.: 

Hero/index.jsx
```
...
import "./Hero.css"
...
```